src/checkbox/src/Checkbox.js

```diff
@@ -35,6 +34,42 @@ MinusIcon.propTypes = {
 
 const noop = () => {}
 
+const pseudoSelectors = {
+  _base: '& + div',
```

Comment:
What does `_base` mean?

Reply:
`_base` is the target element for the styles to get applied to: notice that we put the classname on the actual input to do a lot of the work around hiding the default DOM checkbox. Otherwise, all of these styles would go to the DOM `input`, which isn't what we want.

Reply:
Could we call it _displayElement or something a little more descriptive?